lib/dialects/mysql/query-generator.js

```diff
@@ -434,6 +435,24 @@ class MySQLQueryGenerator extends AbstractQueryGenerator {
     return result;
   }
 
+  selectFromTableFragment(options, model, attributes, tables, mainTableAs) {
```

Comment:
No need to override this you can implement this with `dialect.supports`.

1. First start with MySQL, define `indexHints: true` someone where [here](https://github.com/sequelize/sequelize/blob/2c18e084509469c7e8b9fdbabcb4c713312c216d/lib/dialects/mysql/index.js#L43)
2. Along with `options.indexHints` you will now also check for `this._dialect.supports.indexHints`

Reply:
Done in bbd3c2d5dd5aced953bb6b50ca00f2ec8191ba7c